plugins/ipc/templates/records/infection_alert.html

```diff
@@ -0,0 +1,13 @@
+[[ item.category ]] [[ item.trigger_datetime | displayDateTime ]]
+Lab #: [[ item.lab_test_id ]]
+<span ng-show="item.seen">
+     Seen
+   <br />
+</span>
+<span ng-hide="item.seen">
+  Unseen
+</span>
+<span ng-show="item.comments">
+   <br />
```

Comment:
If a patient is seen they have a `<br>` after but if there are comments they have a `<br>` before, I think you just want `Unseen <br />`